domains/transfer-agreements/TransferAgreements.API/API/Program.cs

```diff
@@ -145,17 +157,17 @@
         };
     });
 
+builder.Services.AddHostedService<DummyMetricsWorker>();
+
 var app = builder.Build();
 
 app.MapHealthChecks("/health");
 
-app.UseOpenTelemetryPrometheusScrapingEndpoint();
-
 app.UseSwagger(o => o.RouteTemplate = "api-docs/transfer-agreements/{documentName}/swagger.json");
 if (app.Environment.IsDevelopment()) app.UseSwaggerUI(o => o.SwaggerEndpoint("/api-docs/transfer-agreements/v1/swagger.json", "API v1"));
 
 app.UseHttpsRedirection();
-JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Clear();
+JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Clear(); //TODO: Why is this needed?
```

Comment:
I know, I added this TODO... but why is it there?

Reply:
Not sure what this is

Reply:
@sahma19 ?

Reply:
Fra @sahma19 "Ja så åbenbart, har vi til vores jwt tokens valgt nogle navne til vores claims, som clasher med claims som dotnet åbenbart automatisk sætter når man genererer jwts. Så som jeg husker det gør det at de claim names, såsom sub, ikke er tilgengælige, også kan vi ikke bruge dem, eller assigne dem nogen værdi, fordi systemet ser det som at vores eget sub claim, ikke eksisterer. "

Tests bliver røde, hvis ikke den er sat

Reply:
"så derfor clearer man den default inbound claim type, for at viske tavlen ren, så der ikke er claim names som allerede er taget, og derfor utilgengælige"